DeviceAdapters/DemoCamera/DemoCamera.cpp

```diff
@@ -1289,6 +1301,47 @@ int CDemoCamera::OnTestProperty(MM::PropertyBase* pProp, MM::ActionType eAct, lo
 
 }
 
+void CDemoCamera::SlowPropUpdate(std::string leaderValue)
```

Comment:
Yes, passing the value as a parameter like this is more correct than accessing `asyncLeader_` (which may have changed in the interim).